overrides/scripts/AE2.zs

```diff
@@ -6,12 +6,22 @@ import mods.gregtech.recipe.RecipeMap;
 <wct:magnet_card>.addTooltip(format.red("This is being removed. A temperary conversion recipe has been added. Check JEI for more details."));
 
 // Tooltips
-<appliedenergistics2:material:59>.addTooltip(format.yellow("Used with the Quantum Ring multiblock. Allows wireless access from anywhere. Check the Quest Book for more information."));
+<appliedenergistics2:material:59>.addTooltip(format.yellow("Used with the Quantum Ring multiblock. Allows wireless access from anywhere. Check the Quest Book for more information.")); //quantum link card
+<appliedenergistics2:material:58>.addTooltip(format.yellow("Used in interfaces. Allows you to put more patterns in an interface then normally allowed up to 4x the normal amount of patterns. Check the Quest Book for more information.")); //pattern expansion card
+<appliedenergistics2:material:53>.addTooltip(format.yellow("Allows machines to craft items as they are used up automatically.")); //crafting card
+<appliedenergistics2:material:30>.addTooltip(format.yellow("Speeds up various ae2 processes such as exporting things into a machine, speeding up molecular accelerators.")); //acceleration card
+<appliedenergistics2:material:60>.addTooltip(format.yellow("When put in a wireless terminal, it will pick items off the floor and put them into your ae2 system based on your configuration.")); //magnet card
+<appliedenergistics2:material:31>.addTooltip(format.yellow("Inverts ae2 configurations. Example: If you have a storage bus set to only store Omnium Ingots, and you have this card, it will store every item but Omnium Ingots")); //inverter card
```

Comment:
Capitalize AE2 and remove the examples. Examples should be put in the qb (doesn't need to be added in this PR)

Reply:
Fixed it

Reply:
can you merge it now